xinference/core/worker.py

```diff
@@ -640,50 +647,55 @@ async def launch_builtin_model(
         assert model_uid not in self._model_uid_to_model
         self._check_model_is_valid(model_name, model_format)
 
-        subpool_address, devices = await self._create_subpool(
-            model_uid, model_type, n_gpu=n_gpu, gpu_idx=gpu_idx
-        )
+        if self.check_model_launch_status(model_uid) is not None:
```

Comment:
`check` seems an action without return value. Maybe can call it get?

Reply:
OK